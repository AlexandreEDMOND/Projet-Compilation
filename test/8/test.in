int main()
{
  float a = -6.5;
  float g = 1856.75982;
  float c;

  int d = -86;
  int e = 69;
  int f;

  c = a;
  a = g;

  f = d;
  d = e;

  print(a);
  printf("\n");
  print(g);
  printf("\n");
  print(c);
  printf("\n");
  print(d);
  printf("\n");
  print(e);
  printf("\n");
  print(f);
  printf("\n");
  
  return 0;
}