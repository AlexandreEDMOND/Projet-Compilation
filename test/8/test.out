1856.75982
1856.75982
-6.5
69
69
-86
